data/moveset.json

```diff
@@ -1,33 +1,33 @@
 {
-	"venusaur": "Solar Beam | Petal Blizzard | Synthesis | Vine Whip",
-	"charizard": "Normal : Inferno | Slash | Flame Burst | Dragon Claw\nMega X : Flare Blitz | Dragon Claw | Shadow Claw \nMega Y : Flamethrower | Dragon Claw | Air Slash | Shadow Claw",
-	"blastoise": "Hydro Pump | Flash Cannon | Skull Bash | Bite",
+	"venusaur": "Solar Beam | Petal Blizzard | Synthesis | Double-Edge",
+	"charizard": "Normal : Flamethrower | Air Slash | Flare Blitz | Dragon Claw\nMega X : Flare Blitz | Dragon Claw | Crunch | Wing Attack\nMega Y : Flamethrower | Dragon Claw | Air Slash | Crunch",
+	"blastoise": "Hydro Pump | Flash Cannon | Skull Bash | Aqua Jet* \n[*: Mega only]",
 	"butterfree": "Air Slash | Bug Buzz | Gust | Psybeam",
-	"beedrill": "Bug Bite(Weedle) | Poison Jab | Assurance | Fell Stinger",
-	"pidgeot": "Hurricane | Air Slash | Twister | Roost",
-	"raticate": "Crunch | Double-edge | Sucker Punch | Hyper Fang \n(same For Alolan Raticate)",
-	"fearow": "Drill Peck | Drill Run | Assurance | Roost",
-	"arbok": "Gunk Shot | Crunch | Ice Fang | Fire Fang",
-	"raichu": "Thunderbolt | Slam | Thunder | Quick Attack",
-	"sandslash": "Earthquake | Slash | Fury Cutter | Rollout",
-	"nidoqueen": "Earth Power | Superpower | Poison Fang* | Crunch* \n[* : nidorina]",
-	"nidoking": "Thrash | Megahorn | Earth Power | Poison Jab* \n[* : nidoran]",
-	"clefable": "Moonblast | Moonlight | Wake-up Slap | Meteor Mash",
-	"ninetales": "Fire Blast* | Flamethrower* | Hex* | Extrasensory* \n[*: Vulpix]",
-	"vileplume": "Acid*| Moonblast* | Solar Beam | Giga Drain* \n[*:Gloom] ",
-	"dugtrio": "Earthquake | Tri Attack | Sucker Punch | Night Slash",
-	"arcanine": "Flare Blitz | Outrage/Play Rough | Extreme Speed | Crunch",
-	"poliwrath": "Dynamic Punch | Mud Bomb* | Hydro Pump* | Body Slam or Wake-Up Slap* \n[*: poliwag or Poliwhirl]",
-	"alakazam": "Future Sight | Recover | Psychic | Psybeam",
-	"machamp": "Dual Chop | Cross Chop | Knock Off | Dynamic Punch",
-	"tentacruel": "Hydro Pump | Poison Jab | Sludge Wave | Hex",
-	"golem": "Stone Edge | Explosion | Earthquake | Steamroller",
-	"rapidash": "Fire Blast | Bounce | Megahorn | Poison Jab",
-	"dodrio": "TriAttack | Drill Peck | Jump Kick | Thrash",
-	"cloyster": "Ice Shard | Razor Shell | Ice Beam* | Hydro Pump \n[* : Shellder]",
+	"beedrill": "Poison Jab | Outrage | Assurance* | Bug Bite* \n[*: Prior to Gen 9 only]",
```

Comment:
What does 'prior to Gen 9 only' means?

Reply:
These moves were available to be learned prior to the move changes with Generation 9. For example, Arbok could learn Gunk Shot back then. It cannot now, but the old mons still have that move and can still use it. 

If you'd prefer to exclude those moves, I can adjust and just have the new moves. That would leave something like Beedrill with only 2 move options though